_src/docs.css

```diff
@@ -121,9 +121,14 @@ a:hover,
 }
 navigation {
```

Comment:
Referencing #30, could we auto-scroll the menu to the active item?

Reply:
Can you clarify?

Reply:
@alexroan Can we land this as a separate feature / PR?